src/app/_components/TagComponents/TagGroupContainer.tsx

```diff
@@ -0,0 +1,7 @@
+type TagGroupContainerProps = {
+  children: React.ReactNode
+}
+
+export function TagGroupContainer({ children }: TagGroupContainerProps) {
+  return <span className="flex gap-2">{children}</span>
+}
```

Comment:
I find the naming of `TagGroupContainer` and `TagGroup` a bit confusing - they seem related, but one serves a layout purpose and the other one layout and logic? Maybe we name `TagGroupContainer` -  `TagWrapper` or `TagLayout`.

Reply:
The thing is - They are related - It's a container for a `TagGroup` 😕

Maybe `TagGroup` can be renamed to `SimpleTagGroup`?

Reply:
But it can also be used just for labels?
 ```js
<TagGroupContainer>
   <TagLabel>
         {getCategoryLabel({ collectionName: 'event_entries', category })}
   </TagLabel>
          {eventHasConcluded && (
   <TagLabel variant="callout">Past Event
   </TagLabel>
          )}
</TagGroupContainer>
```

Reply:
`TagGroupContainer` is wrapping two `TagLabels` in this case.